lib/ranked-model.rb

```diff
@@ -42,6 +43,10 @@ def ranker name
       end
     end
 
+    def ranker_connection_class
+      ActiveRecord::Base
```

Comment:
We explicitly rely on ActiveRecord so why do we need to abstract this? :)

Reply:
for application with multiple databases it's nice to be able to specify the connection class.  ideally really it would just be the current active record class which would always do the right thing due to the way the connection handler threading is handled, but having a way to easily specify the connection class is probably best in both cases

Reply:
We may not need this per my other comment below.